### ISSUE TITLE
Could you please put pretrained model again?

### ISSUE THREAD

**Issue by Arlen0615** [CPFLAME/LOCO#3]
Hi CPFLAME,

the link of model is gone
"For testing We released our pretrained model at **model**, you can download it for testing."

Could you please put pretrained model again? thanks very much.

**Comment by CPFLAME** [comment]
@Arlen0615  Hi Arlen，
Sorry for my late reply, 
The link is https://pan.baidu.com/s/1pKKtKh1, and I can visit it successfully. Can you visit this link?

**Comment by Arlen0615** [comment]
@CPFLAME 
No.. I can not visit, after I click the link, it show error page https://pan.baidu.com/error/404.html.
